lib/test-handler/helpers/describe.js

```diff
@@ -0,0 +1,101 @@
+const { api } = require('../../api');
+const { config } = require('../../config/config');
+const {
+    reportException,
+    getTestKey,
+    formatStatus,
+    getRelativePath,
+    getStatus,
+    formingSteps,
+} = require('./common');
+
+/**
+ * Separate test key to key and version.
+ *
+ * @param {string} testKey - Test key.
+ * @returns {{version: string, key: string}} Separated key and version.
+ */
+const getTestKeyVersion = testKey => {
+    const result = RegExp(/(.*?)(?:\((.*)\))/).exec(testKey);
```

Comment:
You also need to describe an example of the name of the test with the key and version in README

Reply:
And as I see you changed the template to describe the key, what was it done for? I think to add a version you had to expand the existing template.